baklava/package.json

```diff
@@ -30,6 +30,7 @@
   "license": "MIT",
   "devDependencies": {
     "@types/node": "^14.14.31",
+    "builder-utils": "^1.0.4",
```

Comment:
Wait a minute what exactly is `builder-utils`?

Reply:
The [npm page](https://www.npmjs.com/package/builder-utils) for it leads to [this url](https://bitbucket.org/mangomap/builder-feature-highligh) that 404s

Reply:
So according to their [npm page](https://www.npmjs.com/package/builder-util) it includes the multiple utils that are required by the electron-builder to build apps. When I tried building an appimage by adding it into the linux.target array, it gave an error for me because builder utils was not available. So I installed and it created an appimage when I run `npm run compile && electron-builder --linux`. 
It basically provides some utils for electron-builder.

Reply:
Pretty sure this is the [builder-util](https://www.npmjs.com/package/builder-util) you are looking for, its by electron-builder's authors also the one you're using rn seems malicious to me

Reply:
Ohh wait its my fault. I installed `builder-utils` instead of `builder-util`.
Fixed it!
(I know that was a serious mistake, sorry)

Reply:
Here is why i think its malicious 
![image](https://user-images.githubusercontent.com/35400192/111900537-35cd7a80-8a44-11eb-917d-eb1a16d896b1.png)

Reply:
> 
> 
> Ohh wait its my fault. I installed `builder-utils` instead of `builder-util`.
> Fixed it!
> (I know that was a serious mistake, sorry)

No problems.